File: Java/level 2/Harshad number.java
public class HarshadNumber{
	public boolean isHarshad(int num){
		String tem = Integer.toString(num);
    String[] arr = tem.split("");
    int nanu=0;
    for(int a=0; a<arr.length;a++){
    	nanu+=Integer.parseInt(arr[a]);
    }
		return (num%nanu==0)?true:false;
	}
  
       // 아래는 테스트로 출력해 보기 위한 코드입니다.
	public static void  main(String[] args){
		HarshadNumber sn = new HarshadNumber();
		System.out.println(sn.isHarshad(18));
	}
}

File: Javascript/level 2/Harshad number.js
function Harshad(n){
  //함수를 완성하세요
	var tem = n.toString();
  var arr = tem.split("");
  var nanu=0;
  for(var a=0; a<arr.length;a++){
    	nanu+=parseInt(arr[a]);
    }
	return (n%nanu==0)?true:false;
}

// 아래는 테스트로 출력해 보기 위한 코드입니다.
console.log(Harshad(18))